x-pack/plugins/apm/server/routes/time_range_metadata/route.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { toBooleanRt } from '@kbn/io-ts-utils';
+import * as t from 'io-ts';
+import { setupRequest } from '../../lib/helpers/setup_request';
+import { getIsUsingServiceDestinationMetrics } from '../../lib/helpers/spans/get_is_using_service_destination_metrics';
+import { createApmServerRoute } from '../apm_routes/create_apm_server_route';
+import { kueryRt, rangeRt } from '../default_api_types';
+
+export const timeRangeMetadataRoute = createApmServerRoute({
+  endpoint: 'GET /internal/apm/time_range_metadata',
```

Comment:
For transaction metrics we handle on the route level for each endpoint. In this case we send `isUsingServiceDestinationMetrics` to the client so it can pass it in every request. Do we eventually want the same behaviour for transaction metrics?

Reply:
Yeah, that's the idea, to move that here as well. Possibly some other stuff too, but not sure yet what.